Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -358,10 +358,16 @@ define([
         content._rtcCenterTransform = Matrix4.clone(Matrix4.IDENTITY);
         var rtcCenter = featureTable.getGlobalProperty('RTC_CENTER', ComponentDatatype.FLOAT, 3);
         if (defined(rtcCenter)) {
-            content._rtcCenterTransform = Matrix4.fromTranslation(Cartesian3.fromArray(rtcCenter), content._rtcCenterTransform);
+            content._rtcCenterTransform = Transforms.eastNorthUpToFixedFrame(Cartesian3.fromArray(rtcCenter), content._rtcCenterTransform);
+            tile._hasRTC = true;
```

Comment:
Here and throughout, object properties, even if private, should be defined in the constructor.

Reply:
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#constructor-functions